stepconf/stepconf.go

```diff
@@ -160,6 +172,15 @@ func setField(field reflect.Value, value, constraint string) error {
 		field.SetFloat(f)
 	case reflect.Slice:
 		field.Set(reflect.ValueOf(strings.Split(value, "|")))
+	case reflect.Ptr:
+		// if the field is a ptr type then we need to initialize a pointer address first
```

Comment:
Would be nice to have  a unit test for setField function(s).

Reply:
![Screen Shot 2019-11-08 at 11 22 13 AM](https://user-images.githubusercontent.com/1564933/68469254-17355d80-021a-11ea-8a49-35b259b87698.png)

it is +95% covered.